bpf/kmesh/probes/probe.h

```diff
@@ -61,10 +61,38 @@ static inline void observe_on_close(struct bpf_sock *sk)
 
     storage = bpf_sk_storage_get(&map_of_sock_storage, sk, 0, 0);
     if (!storage) {
-        BPF_LOG(ERR, PROBE, "close bpf_sk_storage_get failed\n");
+        __u32 src_ip4 = sk->src_ip4;
```

Comment:
Previously it was found that the src and dst of inbound traffic is reversed with outbound traffic, don't we need to consider this over here?